.circleci/config.yml

```diff
@@ -26,7 +26,64 @@ e2eTest: &e2eTest
         command: ./e2e-harness teardown
         when: always
 
-version: 2
+version: 2.1
+
+orbs:
+  app-catalog:
+    executors:
+      my-executor:
+        docker:
+          - image: circleci/golang:1.12.4
+  
+    commands:
+      create-tarball:
+        steps:
+          - run:
+              name: "Creates a tarball file"
+              command: |
+                set -o errexit
+                set -o nounset
+                set -o pipefail
+      
+                readonly HELM_DIR=helm/${CIRCLE_PROJECT_REPONAME}-chart
+                readonly HELM_URL=https://storage.googleapis.com/kubernetes-helm
+                readonly HELM_TARBALL=helm-v2.11.0-linux-amd64.tar.gz
```

Comment:
I'd prefer to:

* Have variables defined below `echo_*` functions as this part is just copy/paste and I'd like to skip that while reading scripts.
* Have scripts variables in lower case. This is to distinguish them from environment variables. This is bash good practice.

Reply:
I feel like they are constants more than scripts variables so it would make sense to keep them capitalized if so.